.github/workflows/release.yml

```diff
@@ -93,17 +92,33 @@ jobs:
             }
             core.setFailed(`Draft release not found`)
 
-      - name: tar CRDs
-        run: |
-          tar -czf CRDS.tar.gz -C config/crd $(ls config/crd)
+      - name: Download SBOM artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3
+        with:
+          name: sbom
+
+      - name: Download CRDs artifact
+        uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3
+        with:
+          name: CRDS
+
+      - name: Display structure of downloaded files
+        run: ls -R
 
       - name: Upload release assets
         id: upload_release_assets
         uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6
         with:
           script: |
             let fs = require('fs');
-            let files = ['audit-scanner-sbom.spdx', 'audit-scanner-sbom.spdx.cert', 'audit-scanner-sbom.spdx.sig', "CRDS.tar.gz"]
+            let files = [
+              'audit-scanner-sbom-amd64.spdx',
+              'audit-scanner-sbom-amd64.spdx.cert',
+              'audit-scanner-sbom-amd64.spdx.sig',
+              'audit-scanner-sbom-arm64.spdx',
+              'audit-scanner-sbom-arm64.spdx.cert',
+              'audit-scanner-sbom-arm64.spdx.sig',
+              "CRDS.tar.gz"]
```

Comment:
I suppose that the release generated by this change is the [`v0.0.1`](https://github.com/viccuad/audit-scanner/releases/tag/v0.0.1) not the [`v0.0.0-test` ](https://github.com/viccuad/audit-scanner/releases/tag/v0.0.0-test)as mentioned in the PR description. Am I right? I'm asking these because I noticed that these files are not uploaded to the `v0.0.0-test` version.

Reply:
yep, you are right. `v0.0.0-test` failed because there wasn't a preexisting GH release, as the job gets the latest previous one. `v0.0.1` is ok.